discord/abc.py

```diff
@@ -315,6 +315,8 @@ def created_at(self):
     def overwrites_for(self, obj):
         """Returns the channel-specific overwrites for a member or a role.
 
+        If no overwrites are set, returns None.
```

Comment:
I feel this should probably be merged with the above sentence, e.g.
```
Returns the channel-specific overwrites for a member or role, or None if no overwrites are set.
```
It might not be necessary at all, however, since it's mentioned in the return signature.

Reply:
Changed per suggestion